R/blastula_email_format.R

```diff
@@ -74,23 +76,37 @@ blastula_email <- function(content_width = "1000px",
 
   # Note bash escaping is handled by R Markdown (via shQuote())
   pandoc_args <- paste0("--variable=content-width:", content_width)
+
   if (isTRUE(connect_footer)) {
-    pandoc_args <- c(
-      pandoc_args,
-      "--variable=rsc-footer:1",
-      paste0(
-        "--variable=rsc-date-time:", add_readable_time(time = Sys.time())),
-      paste0(
-        "--variable=rsc-report-url:",
-        Sys.getenv("RSC_REPORT_URL", unset = "https://connect.example.com/content/1234/")),
-      paste0(
-        "--variable=rsc-report-rendering-url:",
-        Sys.getenv("RSC_REPORT_RENDERING_URL", unset = "https://connect.example.com/content/1234/_rev5678")),
-      paste0(
-        "--variable=rsc-report-subscription-url:",
-        Sys.getenv("RSC_REPORT_SUBSCRIPTION_URL", unset = "https://connect.example.com/connect/#/apps/1234/subscriptions")),
-      paste0("--variable=rsc-report-name:", Sys.getenv("RSC_REPORT_NAME"))
-    )
+
+    pandoc_args <-
+      c(
+        pandoc_args,
+        "--variable=rsc-footer:1",
+        paste0(
+          "--variable=rsc-date-time:", add_readable_time(time = Sys.time())
+        ),
+        paste0(
+          "--variable=rsc-report-url:",
+          Sys.getenv(
+            "RSC_REPORT_URL",
+            unset = "https://connect.example.com/content/1234/"
+          )
+        ),
+        paste0(
+          "--variable=rsc-report-rendering-url:",
+          Sys.getenv(
+            "RSC_REPORT_RENDERING_URL",
+            unset = "https://connect.example.com/content/1234/_rev5678")
+        ),
+        paste0(
+          "--variable=rsc-report-subscription-url:",
+          Sys.getenv(
+            "RSC_REPORT_SUBSCRIPTION_URL",
+            unset = "https://connect.example.com/connect/#/apps/1234/subscriptions")
+        ),
+        paste0("--variable=rsc-report-name:", Sys.getenv("RSC_REPORT_NAME"))
+      )
```

Comment:
Just so you know, if this can help make your code easier to maintain, **rmarkdown** has `rmarkdown::pandoc_variable_arg()` you can use to build `--variable` argument. 

It would avoid all the `paste0()`

Reply:
I'm going to try using that `rmarkdown` function in a different PR. Thanks for letting me know!